Kata RPG/Kata RPG/Test.swift

```diff
@@ -79,12 +79,51 @@ func thirdIteration(){
 }
 
 func priestAttack(){
-    p1.attack(target: w1)
+    Zelda.attack(target: Ganon)
 }
 
 func priestHeal(){
-    p1.health = 5
-    print(p1)
-    p1.heal(target: p1)
-    print(p1)
+    Zelda.health = 5
+    print(Zelda)
+    Zelda.heal(target: Zelda)
+    print(Zelda)
+}
+
+
+
+let Hyrule = Faction(name: "Hyrule")
+let Zebes = Faction(name: "Zebes")
+let Samus = Warrior(name: "Samus")
+
+func fourthIteration(){
+    attackAnotherFaction()
+    healAnotherFaction()
+}
+
+func attackAnotherFaction(){
+    print(Link,Ganon)
+    Link.joinFaction(faction: Hyrule)
+    Ganon.joinFaction(faction: Hyrule)
+    Link.attack(target: Ganon)
+    print(Link,Ganon)
+}
```

Comment:
How do I know it didn't work ? Aren't characters from the same faction supposed to not hurt eachother?